packages/config/src/bin/prod.webpack.config.ts

```diff
@@ -18,7 +18,7 @@ const { config: webpackConfig, plugins } = config({
 plugins.push(...commonPlugins, ...externalPlugins);
 
 const start = (env: { analyze?: string }): Configuration => {
-  if (env && env.analyze === 'true') {
+  if (BundleAnalyzerPlugin && env && env.analyze === 'true') {
```

Comment:
I think we can just kick it out entirely. Plugins can be extended from within `fec.config.js` anyway. Can you add a deprecation warning to this condition? So we can remove it in next major release?